README.md

```diff
@@ -1,6 +1,69 @@
 # curriculum-cr-training-mobile-menu
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Project Name
```

Comment:
- [ ] Please, add your project name here because this is the title for your prject